package.json

```diff
@@ -14,7 +14,7 @@
   "scripts": {
     "test": "./node_modules/.bin/mocha --reporter spec --recursive",
     "test watch": "./node_modules/.bin/mocha --reporter spec --recursive --watch",
-    "cover": "./node_modules/istanbul/lib/cli.js cover _mocha test/* --reporter --recursive",
+    "cover": "./node_modules/istanbul/lib/cli.js cover _mocha test/* test/**/**/* --reporter --recursive",
```

Comment:
Change this to `./node_modules/istanbul/lib/cli.js cover _mocha --recursive \"test/**/*_spec.js\" --reporter spec`